setup.sh

```diff
@@ -259,12 +259,6 @@ COPYLEFT_LICENSE_INCLUDE = 'GPL* LGPL*'
   fi
 
   case $my_sdk_host in
-    win32)
-      extra_conf="SDKMACHINE = \"i686-mingw32\""
-      ;;
-    win64)
```

Comment:
Does anybody try to build it under WSL2? I am just curious...

Reply:
It should be possible https://docs.yoctoproject.org/5.0/singleindex.html#setting-up-to-use-windows-subsystem-for-linux-wsl-2

But with headaches...